### PR TITLE
Update ethnicity question

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?
Updated content in all ethnicity related questions for census household

Card: https://trello.com/c/QiXJynsS/1489-update-ethnicity-s

### How to review 
Run SR and check content updated and all tests pass

### CODE REVIEW

File: data/en/census_household.json [hunk 1]
```diff
@@ -1833,22 +1833,22 @@
...
-                                        "description": "Including English / Welsh / Scottish / Northern Irish / British, Irish, Gypsy or Irish Traveller or any other White background"
+                                        "description": "Including English, Welsh, Scottish, Northern Irish, British, Irish, Gypsy, Irish Traveller or any other White background"
                                     },
                                     {
-                                        "label": "Mixed / multiple ethnic groups",
-                                        "value": "Mixed / multiple ethnic groups",
-                                        "description": "Including White and Black Caribbean, White and Black African,  White and Asian or any other Mixed/multiple ethnic background"
+                                        "label": "Mixed or multiple ethnic groups",
+                                        "value": "Mixed or multiple ethnic groups",
+                                        "description": "Including White and Black Caribbean, White and Black African, White and Asian or any other Mixed/multiple ethnic background"
```
Comment (shahi645) [inline data/en/census_household.json]: @MIslam92  shouldn't this be `Mixed or multiple`?

Comment (MIslam92) [reply data/en/census_household.json]: Thats a content thing all the card said was to replace / with or etc.

Comment (shahi645) [reply data/en/census_household.json]: @MIslam92 it is a `/` ? so should be replaced? you have on other descriptions in this PR

**Review by shahi645 — APPROVED** [review]

**Comment by shahi645** [comment]
My bad, @MIslam92 you used the wrong jinja filter for the name

**Review by dwyeradam — APPROVED** [review]
@MIslam92  Squishy Squashy.

**Pull request merged**